relate/src/main/scala/com/lucidchart/relate/SqlQuery.scala

```diff
@@ -79,6 +79,22 @@ trait Sql {
    */
   def results()(implicit connection: Connection): ResultSet = normalStatement.results()
 
+  /**
+    * Provides a java.sql.ResultSet that streams records from the database.
+    * This allows for interacting with large data sets with less risk of OutOfMemoryErrors.
+    * Many JDBC connectors will not allow for additional queries to the connection until the
+    * returned ResultSet has been closed.
+    * @param fetchSize the number of rows to fetch at a time, defaults to 100. If the JDBC Driver
```

Comment:
Where does this default come from? It seems like the fetchSize is required here.